src/ado2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -158,6 +158,12 @@ public async Task Invoke(GenerateScriptCommandArgs args)
             _adoInspectorService.OrgFilter = args.AdoOrg;
             _adoInspectorService.TeamProjectFilter = args.AdoTeamProject;
 
+            if (await _adoInspectorService.GetRepoCount() == 0)
+            {
+                _log.LogError("A migration script could not be generated because no migratable repos were found. Please note that the GEI does not migrate disabled and TFVC repos.");
```

Comment:
```suggestion
                _log.LogError("A migration script could not be generated because no migratable repos were found. Please note that the GEI does not migrate disabled or TFVC repos.");
```